2d/benchmarks/dambreak_Colagrossi/test_dambreak_Colagrossi.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+import pytest
+import unittest
+from proteus.iproteus import *
+from proteus import Comm
+comm = Comm.get()
+import dambreak_Colagrossi_so
+import os
+from numpy import *
+from scipy import *
+from pylab import *
+import collections as cll
+import csv
+
+class TestDambreakCollagrossiTetgen(unittest.TestCase):
+
+    @classmethod
+    def setup_class(cls):
+        pass
+
+    @classmethod
+    def teardown_class(cls):
+        pass
+
+    def setup_method(self,method):
+        pass
+
+    def teardown_method(self,method):
+        """ Tear down function """
+        FileList = ['dambreak_Colagrossi.xmf',
+                    'dambreak_Colagrossi.h5']
+        for file in FileList:
+            if os.path.isfile(file):
+                os.remove(file)
+            else:
+                pass
+            
+    def test_run(self):
+        from petsc4py import PETSc
+        pList = []
+        nList = []
+        for (p,n) in dambreak_Colagrossi_so.pnList:
+            pList.append(__import__(p))
+            nList.append(__import__(n))
+            if pList[-1].name == None:
+                pList[-1].name = p
+        so = dambreak_Colagrossi_so
+        so.name = "dambreak_Colagrossi"
+        if so.sList == []:
+            for i in range(len(so.pnList)):
+                s = default_s
+                so.sList.append(s)
+        Profiling.logLevel=7
+        Profiling.verbose=True
+        # PETSc solver configuration
+        OptDB = PETSc.Options()
+        with open("../../../inputTemplates/petsc.options.asm") as f:
```

Comment:
@cekees @lmaurel  - I just pushed a branch  [https://github.com/erdc-cm/proteus/pull/506](url) with some new features in the TestTools module.  There are two relevant features for the `air-water-vv repo` tests:  
First, there is a function `_setPETSc` which given a petsc options file sets the petsc options (much as what you've done above Chris).
Second, there is a special class `AirWaterVVTest` that includes a function `getPETScDir` which will should set the relative path for the Petsc input files.
Any comments and ideas would be welcome but I think this is one approach we could follow.